asset_stream_manager/session.cc

```diff
@@ -51,7 +51,7 @@ Session::Session(std::string instance_id, std::string instance_ip,
                    /*forward_output_to_logging=*/true),
       metrics_recorder_(std::move(metrics_recorder)) {
   assert(metrics_recorder_);
-  remote_util_.SetIpAndPort(instance_ip, instance_port);
+  remote_util_.SetHostAndPort(instance_ip, instance_port);
```

Comment:
Can you change `instance_ip` to `instance_host` to match the semantic?

Reply:
I'll do this in a follow-up CL. I didn't want to touch AS30 yet unless necessary.